### DISCUSSION TITLE
Apex chart squashed in mobile view

### DISCUSSION THREAD

**Discussion by LemonPie22** [filamentphp/filament#11251] [Help]
### Package

Other

### Package Version

Filament v3.2.15, Apex charts v3.1.2

### How can we help you?

I've tried to make my chart responsive as shown below but it keeps coming out squashed in mobile view.
```
'chart' => [
    'type' => 'bar',
    'stacked' => true,
    'responsive' => [
        'breakpoint' => '435px',
        'options' => [
            'height' => '300px',
        ],
    ],
],
```

![Screenshot_20240205_105220_Chrome](https://github.com/filamentphp/filament/assets/107488019/290c1a54-a9a1-408f-a278-a09accf62d26)

**Comment by leandrocfe** [comment]
Please open an issue [here](https://github.com/leandrocfe/filament-apex-charts/issues) and share the whole code you are trying to do.

**Reply by LemonPie22** [reply]
Done https://github.com/leandrocfe/filament-apex-charts/issues/62

**Reply by leandrocfe** [reply]
thank you
